ccd-definition/ComplexTypes/C21Order.json

```diff
@@ -0,0 +1,26 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "C21Order",
+    "ListElementCode": "orderTitle",
+    "FieldType": "Text",
+    "ElementLabel": "Order title",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "C21Order",
+    "ListElementCode": "orderDetails",
+    "FieldType": "TextArea",
+    "ElementLabel": "Order details",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "C21Order",
+    "ListElementCode": "c21OrderDocument",
+    "FieldType": "Document",
+    "ElementLabel": "Order PDF",
```

Comment:
is this the right label?

Reply:
this label is never actually displayed anywhere IIRC, but will change to something better